views/index.html

```diff
@@ -306,6 +306,20 @@
 {{/unless}}
 
 <script src="{{static}}/js/vendor/jquery-1.11.0.min.js"></script>
+<script>
+  // Load libraries from a server config file
+  // This method is used so we can both expand vars, and support <script> in the json strings
+  window.libraries = window.libraries || [];
+  $.ajax({
+    url: '{{root}}/{{config.libs.jsLibPath}}',
+    async: false,
+    dataType: 'json',
+    success: function (response) {
+      // Allow for variable replacement
+      window.libraries = window.libraries.concat(JSON.parse(JSON.stringify(response).replace(/#{jsbin.static}/g, "{{static}}")));
```

Comment:
Why is this being double encoded?

Really, *any* relative urls in the libraries should be based off static, all others should point to https or http (in the worst case).